snuba/cli/consumer.py

```diff
@@ -77,7 +77,7 @@
     "--rapidjson-deserialize",
     default=False,
     type=bool,
-    help="Uses rapidjson to deserialize messages",
+    help="[DEPRECATED] It does nothing here",
```

Comment:
Need to remove the usage of this field from the production setup before we remove the CLI parameter.